GitUI/CommandsDialogs/FormBrowse.InitMenusAndToolbars.cs

```diff
@@ -137,17 +137,35 @@ void WorkaroundToolbarLocationBug()
             }
         }
 
+        private void UpdateTooltipWithShortcut(ToolStripItem button, Command command)
+            => UpdateTooltipWithShortcut(button, GetShortcutKeys(command));
+
+        private void UpdateTooltipWithShortcut(ToolStripItem button, Keys keys)
+            => UpdateTooltipWithShortcut(button, keys.ToShortcutKeyToolTipString());
+
+        private void UpdateTooltipWithShortcut(ToolStripItem button, string shortcut)
+        {
+            if (string.IsNullOrWhiteSpace(shortcut))
+            {
+                return;
+            }
+
+            int indexShortcut = button.ToolTipText.LastIndexOf('(');
+            string toolTip = indexShortcut == -1 ? button.ToolTipText : button.ToolTipText.Substring(0, indexShortcut);
+            button.ToolTipText = toolTip.Trim() + shortcut;
```

Comment:
`Trim` is not needed with special separator character. Otherwise, `TrimEnd` suffices.